templates/engage/shared/_de_engage_form.html

```diff
@@ -54,10 +54,10 @@
         <input type="hidden" aria-hidden="true" aria-label="hidden field" name="munchkinId" class="mktoField" value="066-EOV-335" />
         <input name="formid" class="mktoField" value="{{ id }}" type="hidden" aria-label="formid">
         <input name="formVid" class="mktoField" value="{{ id }}" type="hidden" aria-label="formVid">
-        <input type="hidden" name="returnURL" value="{{ returnURL }}" aria-label="returnURL">
-        <input type="hidden" name="retURL" value="{{ returnURL }}" aria-label="retURL">
-        <input type="hidden" name="returnURL" value="{{ returnURL }}" aria-label="return_url">
-        <input type="hidden" aria-hidden="true" aria-label="hidden field" name="return_url" value="{{returnURL}}" />
+        <input type="hidden" name="returnURL" value="{% if ".pdf" in returnURL %}http://ubuntu.com/engage/thank-you?resource_name={{ resouce_name | urlencode }}&resource_url={{ returnURL | urlencode }}{% else %}{{ returnURL }}{% endif %}" aria-label="returnURL">
```

Comment:
You're doing a lot of work here just to pass through the `resource_name`, presumably so you can display it on the thank you page text. Is this really necessary? Couldn't we just say something generic like "your file is now ready to download"?

Reply:
Also, rather than this complex `.pdf`-based logic, why can't we just make the `form_return_url` in the `engage/page.md` frontmatter simply the full URL of the thank-you page?

Reply:
For the first point, because that was what Marketing asked for and it isn' that hard?

For the second point, I didn this because it means we don't have to go and edit all the existing engage pages.  However, the new yaml might mean we don't have to do this at all?